### DISCUSSION TITLE
Cannot find the path specified

### DISCUSSION THREAD

**Discussion by naviprobeha** [microsoft/AL-Go#1026] [Q&A]
Hello!

When running the pipeline on self-hosted runner, since yesterday all repos return this error message when "creating container", right after "creating image":

Error: Unexpected error when running action. Error Message: re-exec error: exit status 1: output: hcsshim::ImportLayer failed in Win32: The system cannot find the path specified. (0x3) ExitCode: 1 Commandline: docker build --isolation=process --memory 8G --no-cache --tag my:sandbox-21.3.51409.62015-se-mt c:\bcartifacts.cache\toxd44kg.04g, StackTrace: at DockerDo, C:\Users\github_service\AppData\Local\Temp\0a987fde-428e-4486-90ed-cbce734fb016\BcContainerHelper\HelperFunctions.ps1: line 272 <- at New-BcImage, C:\Users\github_service\AppData\Local\Temp\0a987fde-428e-4486-90ed-cbce734fb016\BcContainerHelper\ContainerHandling\New-NavImage.ps1: line 711 <- at New-BcContainer, C:\Users\github_service\AppData\Local\Temp\0a987fde-428e-4486-90ed-cbce734fb016\BcContainerHelper\ContainerHandling\New-NavContainer.ps1: line 620 <- at <ScriptBlock>, C:\Users\github_service\AppData\Local\Temp\0a987fde-428e-4486-90ed-cbce734fb016\BcContainerHelper\AppHandling\Run-AlPipeline.ps1: line 798 <- at <ScriptBlock>

I cannot figure this one out. Anyone encountered this?

/Håkan

**Comment by freddydk** [comment]
What OS is your self-hosted runners running?
I have seen this during image creation on self-hosted runners - not after.

**Comment by naviprobeha** [comment]
Hello!

It is windows 2022.
I have 3 selfhosted runners running, at it seems that it is fixed on at least one of them now.
Will try the third one and see if it comes up.

I have cleaned up the downloaded images and docker with

Docker image prune
Docker system prune

**Comment by naviprobeha** [comment]
Correction: 2 runners out of 3 get this error message.

**Comment by freddydk** [comment]
I would probably delete the runners and recreate them, but if you want to try and repair them - maybe try this tool:
https://freddysblog.com/2018/12/11/clean-up-after-yourself-docker-your-mom-isnt-here/

**Comment by naviprobeha** [comment]
One of them is a freshly installed runner. I have also removed the github-runner one more time and re-installed it.

Seem strange that I have it on 2 different runners.

**Comment by naviprobeha** [comment]
This solved the issue. I’m suspecting that the base image was the issue.

**Comment by naviprobeha** [comment]
I was a bit too eager. It seems its a random error. But now occuring on every build.
I have tried the cleaning script repeatably, and also a complete reinstall of Docker itself.

No luck.
